{"sid": "d44b26f921eb40098dee28bba7ea1aa0", "uid": null, "qid": "71e14c9d6d864c4fa2a5265109bc0569", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [71e14c9d6d864c4fa2a5265109bc0569]\nTraceback (most recent call last):\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/action_dispatcher.py\", line 139, in process\n    output = self.workerContext.execution.execute(data, self.ray, self.state, update)\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/application_interface.py\", line 66, in execute\n    self.execution_function(model)\n  File \"/openfabric/app/main.py\", line 92, in execute\n    msg , img , obj , sessid  = agent.Exec(user_prompt)\n  File \"/openfabric/app/Agent/agent.py\", line 72, in Exec\n    text = self.process(response)\n  File \"/openfabric/app/Agent/agent.py\", line 118, in process\n    self.exit(parsed)\n  File \"/openfabric/app/Agent/agent.py\", line 85, in exit\n    Agent.db.save_session(self.session_id,self.username,resp['summary'],Agent.sessions[self.session_id])\n  File \"/openfabric/app/Agent/Storage/VectorDB.py\", line 46, in save_session\n    VectorDB.sessions.insert_one({\"_id\":session_id, \"text\": text,\"summary\":summary})\nAttributeError: type object 'VectorDB' has no attribute 'sessions'\n", "created_at": "2025-06-28T18:59:06.402169"}], "status": "FAILED", "created_at": "2025-06-28T18:58:41.161554", "updated_at": "2025-06-28T18:59:06.402576", "finished": true}